_includes/sites-world-map.html

```diff
@@ -0,0 +1,320 @@
+        shortName: 'ESiWACE',
+        devSite: false,
+        forceLabelCoors: [0, -60],
+        latLonDecimalDegrees: [53.56686, 9.97658],
+        umPartnerSite: false},
+      { name: 'Bureau of Meteorology',
+        shortName: false,
+        devSite: false,
+        forceLabelCoors: [0, -40],
+        latLonDecimalDegrees: [-37.81966, 144.95086],
+        umPartnerSite: true},
+      { name: 'National Centre for Atmospheric Science (NCAS)',
+        shortName: 'NCAS',
+        devSite: false,
+        forceLabelCoors: [0, -40],
+        latLonDecimalDegrees: [53.8045, -1.56147],
+        umPartnerSite: false},
+      { name: 'Korean Meteorological Administration (KMA)',
+        shortName: 'KMA',
+        devSite: false,
+        forceLabelCoors: false,
+        latLonDecimalDegrees: [37.494, 126.917],
+        umPartnerSite: true},
+      { name: 'Plymouth Marine Laboratory',
+        shortName: false,
+        devSite: false,
+        forceLabelCoors: [0, 25],
+        latLonDecimalDegrees: [50.36615, -4.14782],
+        umPartnerSite: false},
+    ];
+    // Colours from the Cylc logo to use for the site markers & labels.
+    const cylcColours = {
+      'red': '#FF5966',
+      'blue': '#00B4FD',
+      'green': '#00C798',
+      'yellow': '#FFCC00',
+    };
+
+    // From array of all sites, create array of all site locations, by merging
+    // co-located sites into their own item to be marked as one.
+    var locationsMap =
+        cylcUsingSiteData.reduce(function (locationAccumulator, currentSite) {
+      var comparison = locationAccumulator[currentSite.latLonDecimalDegrees];
+      if (comparison == null) {
+        comparison = currentSite;
+      } else {
+        currentSite.shortName =
+            currentSite.shortName.concat(' & ', comparison.shortName);
+        currentSite.name = currentSite.name.concat(' & ', comparison.name);
+        comparison = {
+          ...comparison,
+          ...currentSite
+        };
+      }
+      locationAccumulator[currentSite.latLonDecimalDegrees] = comparison;
+      return locationAccumulator;
+    }, {});
+    const cylcUsingSiteDataByLocation =
+      Object.keys(locationsMap).map(function (key) {
+      return locationsMap[key];
+    });
+
+    // Create world map.
+    var map = new ol.Map({
+      target: 'map',
+      layers: [
+        new ol.layer.Tile({
+          source: new ol.source.Stamen({
+            layer: 'watercolor',
+            wrapX: false, // disable continuously-wrapped horiztonal map.
+          })
+        }),
+        new ol.layer.Tile({
+          source: new ol.source.Stamen({
+            layer: 'terrain-labels',
+            wrapX: false, // disable continuous horizontal label re-display.
+          })
+        })
+      ],
+      view: new ol.View({
+        center: ol.proj.fromLonLat([30, 0].reverse()),
+        zoom: 2,
+        minZoom: 2 // prevent zoom out whereby labels ugily overwhelm view
```

Comment:
s/ugily/uglily ?

Reply:
and then, s/uglily/ugly ?  :rofl:

Reply:
Hmm, maybe I will change the word to avoid us trying to work out how to phrase it! :smile: